packages/next-auth/src/nextAuthPlugin.ts

```diff
@@ -3,39 +3,71 @@ import {
   PartialStlContext,
   MakeStainlessPlugin,
   Params,
+  StlContext,
 } from "stainless";
 import { IncomingMessage, ServerResponse } from "http";
 import { NextServerContext } from "@stl-api/next";
 import { AuthOptions, Session, getServerSession } from "next-auth";
 
+/// An authenticated session, with user data available.
```

Comment:
nit, `///` is not idiomatic in JS – I think you want jsdoc here:
```suggestion
/** An authenticated session, with user data available. */
```